From 31f055c8ecf490a9d01d7fc2b0cc6c5d6928cbae Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Fri, 16 Jun 2017 14:32:18 +0800
Subject: [PATCH] lsi-axm55xx: fix sha1.c compile error

Fix compile error as below:

sha1.c:392:19: error: '_sha1_src' defined but not used [-Werror=unused-const-variable=]
 static const char _sha1_src[] = "_sha1_src";
                    ^~~~~~~~~

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 lib/sha1.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/sha1.c b/lib/sha1.c
index a121224..37b1bc0 100644
--- a/lib/sha1.c
+++ b/lib/sha1.c
@@ -389,8 +389,6 @@ void sha1_hmac(const unsigned char *key, int keylen,
 	memset (&ctx, 0, sizeof (sha1_context));
 }
 
-static const char _sha1_src[] = "_sha1_src";
-
 #ifdef SELF_TEST
 /*
  * FIPS-180-1 test vectors
-- 
1.9.1

